src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+    public Event(String description, LocalDateTime from, LocalDateTime to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (from: " + from.format(DateTimeFormatter.ofPattern("d MMM yyyy| hh:mm a")) +
+                " to: " + to.format(DateTimeFormatter.ofPattern("d MMM yyyy| hh:mm a")) + ")";
```

Comment:
Should there be break before an operator instead of break after an operator?
```suggestion
        return "[E]" + super.toString() + " (from: " + from.format(DateTimeFormatter.ofPattern("d MMM yyyy| hh:mm a"))
                + " to: " + to.format(DateTimeFormatter.ofPattern("d MMM yyyy| hh:mm a")) + ")";
```